src/Makefile.am

```diff
@@ -250,8 +250,14 @@ EQUIHASH_TROMP_SOURCES = \
   pow/tromp/equi.h \
   pow/tromp/osx_barrier.h
 
+EQUIHASH_TROMP_AVX2_SOURCES = \
+  pow/tromp/blake2-avx2/blake2b-common.h \
+  pow/tromp/blake2-avx2/blake2b-load-avx2-simple.h \
+  pow/tromp/blake2-avx2/blake2bip.c \
+  pow/tromp/blake2-avx2/blake2bip.h
+
 # crypto primitives library
-crypto_libbitcoin_crypto_a_CPPFLAGS = $(BITCOIN_CONFIG_INCLUDES) -DEQUIHASH_TROMP_ATOMIC
+crypto_libbitcoin_crypto_a_CPPFLAGS = $(BITCOIN_CONFIG_INCLUDES) -DEQUIHASH_TROMP_UNROLL
```

Comment:
ATOMIC is required for multi-threaded mining. The zcash daemon doesn't need this since it instead launches multiple single-threaded instances. UNROLL does some parameter specific optimizations which gain a few percent in performance, so might as well enable.